api/db/roles.js

```diff
@@ -4,18 +4,23 @@ const getAllActiveRoles = async (userRole, { db = knex } = {}) => {
   const roles = await db('auth_roles')
     .select('id', 'name')
     .where({ isActive: true });
-  if (userRole === 'eAPD Federal Admin') {
-    return roles.filter(
-      role =>
-        role.name !== 'eAPD System Admin' && role.name !== 'eAPD Federal Admin'
-    );
+  
+  switch (userRole) {
```

Comment:
I am always wary of doing filtering on results outside of the DB.  The DB is really really good at filtering, we should let it do filtering where we can.  These filters are not super complicated and can definitely be implemented in knex with .whereIn('name', [foo, bar, baz]) 

`
const query =  db('auth_roles')
    .select('id', 'name')
    .where({ isActive: true });

  switch (userRole) {
    case 'eAPD System Admin':
      return query;

`